ultimarc/ui/devices/device.py

```diff
@@ -50,11 +48,11 @@ def __init__(self, args, env, attached,
         self.attached = attached
 
     # Implement function in child class
-    def roleNames(self):
+    def roleNames(self) -> typing.Dict:
         return None
```

Comment:
The return type hint should be "typing.Optional[typing.Dict]" to indicate a None value or a dictionary may be returned.